fe/fe-core/src/main/java/com/starrocks/planner/OlapScanNode.java

```diff
@@ -475,7 +475,7 @@ private void getScanRangeLocations() throws UserException {
 
     private void computeTabletInfo() throws UserException {
         long localBeId = -1;
-        if (Config.enable_local_replica_selection) {
+        if (!Config.use_staros && Config.enable_local_replica_selection) {
```

Comment:
`LakeSystemInfoService` gives a default implementation and no need to check `use_staros`